api/definition/swagger.yaml

```diff
@@ -0,0 +1,273 @@
+openapi: 3.0.0
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/ANDREWJEVANS/fingertips-next-api/1.0.0
+info:
+  description: An API to query public health indicator data.
+  version: "1.0.0"
+  title: Fingertips API
+  contact:
+    email: you@your-company.com
+tags:
+  - name: indicators
+    description: Endpoints dealing with public health indicators
+paths:
+  /indicators:
+    get:
+      tags:
+        - indicators
+      summary: searches indicators
+      operationId: searchIndicators
```

Comment:
Is it going to get confusing if we refer to this as searching indicators? Isn't it more of a filter on a request for all indicators?